README.md

```diff
@@ -113,31 +113,31 @@ Where:
 The robot is encouraged to track $w_z$ reference commanded by the user.
 
 ```math
-R_{ang\_vel} = \exp[-(w^{ref}_{z} - w_{z})^2]
+R_{ang\_vel} = \exp[-(w^{cmd}_{z} - w^{base}_{z})^2]
 ```
 
 Where:
-- $w_{cmd,z}$ is the commanded yaw velocity.
-- $w_{base,z}$ is the actual yaw velocity.
+- $w^{cmd}_{z}$ is the commanded yaw velocity.
+- $w^{base}_{z}$ is the actual yaw velocity.
 
 #### 3. **Height Penalty**
 
 The robot is encouraged to maintain a desired height as specified by the commanded altitude. A penalty is applied for deviations from this target height:
 
 $$
-R_{z} = (z - z_{ref})^2
```

Comment:
This was a bit confusing, as the reward should be bigger when the robot is at the desired height, and smaller when it deviates from it. So, I think, we either need to change the formula (e.g. $\exp[-(z - z_{ref})^2])$ or keep the same formula but change the name of the variable to "penalty" instead of "reward" (basically, what the title suggests). I go with the second option, but I realize that it might add some confusion around the naming of the variables. Let me know what you think.